plugins/source/gitlab/client/client.go

```diff
@@ -2,11 +2,8 @@ package client
 
 import (
 	"context"
-	"fmt"
 
-	"github.com/cloudquery/plugin-pb-go/specs"
-	"github.com/cloudquery/plugin-sdk/v3/plugins/source"
-	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
```

Comment:
this doesn't belong to this PR (gitlab diff shouldn't be present in googleanalytics migration)